packages/internal/src/__tests__/nestedPages.test.ts

```diff
@@ -142,7 +142,7 @@ describe('User specified imports, with static imports', () => {
   loader: () => import("./pages/Jobs/JobsPage/JobsPage")
 }`)
 
-    expect(outputNoStaticImports).not.toContain(`const JobsNewJobPage = {
+    expect(outputNoStaticImports).not.toContain(`const EditJobPage = {
```

Comment:
I don't understand this edit. Why did you change from `JobsNewJobPage` to `EditJobPage`? Now the variable name doesn't go with content of the variable (`name: "JobsNewJobPage"` etc)

Reply:
You're right, typo while copying over